examples/platform/silabs/provision/ProvisionStorageDefault.cpp

```diff
@@ -70,14 +70,29 @@ CHIP_ERROR ErasePage(uint32_t addr)
     return chip::DeviceLayer::Silabs::GetPlatform().FlashErasePage(addr);
 }
 
-CHIP_ERROR WritePage(uint32_t addr, const uint8_t * data, size_t size)
+size_t RoundNearest(size_t n, size_t multiple)
```

Comment:
Same comment about @mkardous-silabs PR, I think there are multiple streams bringing changes in provisioning, might cause conflict

Reply:
It is actually from #36964 PR but I need it with the new sisdk due to psa crypto change in our sdk. I will drop the commit/rebase once that is merge but I could complete a build without it